packages/eslint-plugin/lib/config/rules/jest.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
   // Disallow setup and teardown hooks
   'jest/no-hooks': 'off',
   // Using a callback in asynchronous tests
-  'jest/no-test-callback': 'error',
+  'jest/no-done-callback': 'error',
```

Comment:
https://github.com/jest-community/eslint-plugin-jest/pull/653

Reply:
`jest/no-try-expect` is deprecated in favor of `jest/no-conditional-expect`

Reply:
Should we add those new rules
- `jest/no-interpolation-in-snapshots`
- `jest/no-deprecated-functions`

Reply:
They seem reasonable, good spot